src/components/FilterMenu.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from "react";
+import Box from "@mui/material/Box";
+import InputLabel from "@mui/material/InputLabel";
+import MenuItem from "@mui/material/MenuItem";
+import FormControl from "@mui/material/FormControl";
+import { allLogLevels } from "../schema/level";
+import { Grid, Select, TextField } from "@mui/material";
+
+interface Props {
+  level: number;
+  onLevelChange: (level: number) => void;
+  beamline: string;
+  onBeamlineChange: (beamline: string) => void;
+  application: string;
+  onApplicationChange: (application: string) => void;
+}
+
+const FilterMenu: React.FC<Props> = (props: Props) => {
+  return (
+    <Box sx={{ padding: "1vw" }}>
+      <Grid container spacing={2}>
+        <Grid item xs={1}>
+          <FormControl fullWidth sx={{ marginTop: 2 }}>
```

Comment:
Not really sure if `sx={{ marginTop: 2 }}` is the best way to do this, it was the first thing I tried that worked:

With:
![image](https://github.com/DiamondLightSource/logpanel/assets/29771545/2a488c60-41fb-4d3f-aac2-62aa10303b98)

Without:
![image](https://github.com/DiamondLightSource/logpanel/assets/29771545/f3f2e606-c89c-48e6-87d8-7a684f4f07ad)

@abigailalexander any ideas?

Reply:
It looks like the two Textfield components below both have `margin="normal"` attributes, I tested out setting this instead of `sx={{marginTop: 2 }}` in FormControl and it achieves the same effect. I suggest you either remove the `margin="normal"` from all the components, or add it to FormControl to be consistent.

Reply:
Brilliant, thanks!